test_cases/solidity/private_data_on_chain/odd_even_fixed/odd_even_fixed.sol

```diff
@@ -0,0 +1,79 @@
+/*
+ * @source: https://github.com/yahgwai/rps
+ * @author: Chris Buckland
+ * Modified by Kaden Zipfel
+ */
+
+pragma solidity ^0.5.0;
+
+contract OddEven {
+    enum Stage {
+        FirstCommit,
+        SecondCommit,
+        FirstReveal,
```

Comment:
When player 1 reveals and sets the contract into Stage.SecondReveal then player 2 knows if he loses or wins without calling reveal(). The contract would be stuck in this state and player 2 does not really have an incentive to call reveal(). Would it make sense to introduce a bond for both players that gets refunded when the Stage.Distribution is reached?

Reply:
@KadenZipfel @maurelian WDYT?

Reply:
You're right, thanks for pointing that out. Just submitted a fix.